roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -43,13 +51,12 @@ def tags(self):
         """
         return []
 
-    @classmethod
-    def is_active(cls, task_vars):  # pylint: disable=unused-argument
+    def is_active(self):  # pylint: disable=unused-argument,no-self-use
```

Comment:
Is there a better way to get pylint to shut up about this? Seems like it would be common enough that your base instance method wouldn't use self.